### DISCUSSION TITLE
How to show two Sort option with created Date , one with Old date and one with new date

### DISCUSSION THREAD

**Discussion by vishalgoyal2472** [microsoft-search/pnp-modern-search#4106] [General]
How can we achieve to show two sort option with created date option as below in image, I have added same Created column two times but its only allowing me to click on one.

**Configuration Screen**

<img width="898" alt="image" src="https://github.com/user-attachments/assets/09dd46c8-a38f-49f9-b6f3-2ae4c03a2b0a">

**Search Result screen**

<img width="559" alt="image" src="https://github.com/user-attachments/assets/eff1d546-c926-4054-b3c2-eae264cfed9e">

**Comment by kasperbolarsen** [comment]
Hi

That is not the way the feature is designed. The intension is that you select a field, like Created, and enable user sort.
and then the Up or Down arrow will allow the users to switch the direction as they wants.
![image](https://github.com/user-attachments/assets/4e61d7b7-d850-4445-bbe6-950667588472)
